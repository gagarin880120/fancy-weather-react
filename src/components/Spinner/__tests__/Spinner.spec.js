import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Spinner from '../Spinner';

describe('Spinner', () => {
  describe('component', () => {
    test('should render without crashing', () => {
      const renderer = new ShallowRenderer();
      renderer.render(<Spinner />);
      const result = renderer.getRenderOutput();
      expect(result).toMatchSnapshot();
    });
  });
});
