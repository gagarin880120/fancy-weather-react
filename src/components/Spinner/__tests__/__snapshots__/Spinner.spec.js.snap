// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should render without crashing 1`] = `
<div
  className="wrapper"
>
  <div
    className="spinner"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
